tests/test_schedule.py

```diff
@@ -0,0 +1,75 @@
+from app.models.energy.ScheduleRuleset import ScheduleTypeLimits, DayValue, ScheduleDay, ScheduleRuleset, ScheduleRule
+from app.models.energy.ScheduleFixedInterval import ScheduleFixedInterval
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType, Date, Time
+from app.models.samples.schedule_ruleset import schedule_ruleset, schedule_ruleset_1
+from app.models.samples.schedule_fixed_interval import schedule_fixed_interval, schedule_fixed_interval1
+from copy import copy
+from pydantic import ValidationError
+import pytest
+
+
+def test_schedule_rule():
+    ScheduleRuleset.parse_obj(schedule_ruleset)
+
+
+def test_schedule_rule1():
+    ScheduleRuleset.parse_obj(schedule_ruleset_1)
+
+
+def test_schedule_rule_wrong():
```

Comment:
Each test should ideally check one mistake. This is getting really long. You can break it down into several tests.